product_docs/docs/pem/8/supported_platforms.mdx

```diff
@@ -8,6 +8,13 @@ For information about the platforms and versions supported by PEM, see [Platform
 !!! Note
     Postgres Enterprise Manager 8.3 and later is supported on SLES.
 
-## Supported locales
+## Database compatibility
```

Comment:
I think this should be Postgres compatibility and that we should spell out the product names in the table. I think there is room for that.